aces/Loris
# [Integration Test] Update Imaging Browser filter tests

Updated the testImagingBrowserFiltersAndShowClearButtons test to use the new `_filterTest` method

#### Testing instructions 

Check out Travis for integration test results or use the following command to run the test on terminal:

`npm run tests:integration -- --filter ImagingBrowserTestIntegrationTest::testImagingBrowserFiltersAndShowClearButtons`
